aten/src/ATen/CPUApplyUtils.h

```diff
@@ -354,28 +354,26 @@ inline void CPU_tensor_parallel_apply1(
     int64_t grain_size = internal::TBB_GRAIN_SIZE) {
   if (!_apply_preamble({tensor1}))
     return;
-  if (tensor1.numel() < grain_size) {
-    CPU_tensor_apply1<scalar1>(tensor1, op);
-    return;
-  }
-  auto range = tbb::blocked_range<size_t>(0, tensor1.numel());
   if (tensor1.ndimension() < 8) {
-    tbb::parallel_for(
-        range, [&tensor1, &op](const tbb::blocked_range<size_t> r) {
+    parallel_for(
+        0,
+        tensor1.numel(),
+        grain_size,
```

Comment:
Can you add LLVM-style kwargs here (`/*argument=*/0, ...`, at least for the `0`)?